src/differentials.jl

```diff
@@ -208,7 +213,11 @@ mul_zero(::Any, ::Zero) = Zero()
 #####
 
 """
-TODO
+    DNE()
```

Comment:
Can we just rename this `DoesNotExist`?

Reply:
Maybe maybe not.
It is beyond the scope of this PR